app/src/sharedTest/java/org/oppia/app/administratorcontrols/AppVersionActivityTest.kt

```diff
@@ -80,35 +80,36 @@ class AppVersionActivityTest {
   }
 
   @Test
-  fun testAppVersionActivity_configurationChange_appVersionIsDisplayedCorrectly(){
-    onView(isRoot()).perform(orientationLandscape())
-    onView(
-      withId(
-        R.id.app_version_text_view
-      )
-    ).check(
-      matches(
-        withText(
-          String.format(context.resources.getString(R.string.app_version_name), BuildConfig.VERSION_NAME)
+  fun testAppVersionActivity_configurationChange_appVersionIsDisplayedCorrectly() {
```

Comment:
Why App version test activity is in this PR.

Reply:
Removed